### ISSUE TITLE
Request Entity Too Large for url [imdb_watchlist]

### ISSUE THREAD

**Issue by SJ50** [Flexget/Flexget#2536]
### Expected behaviour:
sync imdb watchlist to local.

### Actual behaviour:
I want to sync imdb watch list to local but I get error "Request Entity Too Large for url"
I get same error where imdb_watchlist is involved. 

### Steps to reproduce:
executing task watchlist_local

#### Config:
```yaml
  watchlist_local:
    priority: 3
    imdb_watchlist:
      user_id: '{? imdb.user_id ?}'
      list: watchlist
    template:
      - reject_documetaries
    seen: local
    list_add:
      - movie_list: Movies
```
 Full config https://pastebin.com/vUpFdQjp
#### Log:

```
2019-12-18 11:39 CRITICAL plugin        watchlist_local 413 Client Error: Request Entity Too Large for url: http://www.imdb.com/title/data?ids=tt5074352%2Ctt0198781%2Ctt5013056%2Ctt4154756%2Ctt1825683%2Ctt6644200%2Ctt2798920%2Ctt5628302%2Ctt2704998%2Ctt1433813%2Ctt5253008%2Ctt7860270%2Ctt1677720%2Ctt5610554%2Ctt7098658%2Ctt6217608%2Ctt4686844%2Ctt6213362%2Ctt5742374%2Ctt5164432%2Ctt1485796%2Ctt3501632%2Ctt2395427%2Ctt0848228%2Ctt2015381%2Ctt3896198%2Ctt2250912%2Ctt2283362%2Ctt3498820%2Ctt5478478%2Ctt2231461%2Ctt1413492%2Ctt1211837%2Ctt4500922%2Ctt2873282%2Ctt2527336%2Ctt0974015%2Ctt0458339%2Ctt1431045%2Ctt0111161%2Ctt0068646%2Ctt0071562%2Ctt0468569%2Ctt0108052%2Ctt0167260%2Ctt0137523%2Ctt0110912%2Ctt0060196%2Ctt0120737%2Ctt0109830%2Ctt0080684%2Ctt1375666%2Ctt0167261%2Ctt0099685%2Ctt0133093%2Ctt0076759%2Ctt0114369%2Ctt0102926%2Ctt0114814%2Ctt0120815%2Ctt0816692%2Ctt1675434%2Ctt0103064%2Ctt0407887%2Ctt0088763%2Ctt0172495%2Ctt0482571%2Ctt0078748%2Ctt1853728%2Ctt2380307%2Ctt0910970%2Ctt1345836%2Ctt0090605%2Ctt0105236%2Ctt0086190%2Ctt0986264%2Ctt1187043%2Ctt0114709%2Ctt0361748%2Ctt0435761%2Ctt0119217%2Ctt2106476%2Ctt1832382%2Ctt1049413%2Ctt0372784%2Ctt0095016%2Ctt0113277%2Ctt2096673%2Ctt0268978%2Ctt3170832%2Ctt0993846%2Ctt0434409%2Ctt0083658%2Ctt1291584%2Ctt0477348%2Ctt0167404%2Ctt0469494%2Ctt0084787%2Ctt0266543%2Ctt0118715%2Ctt0091251%2Ctt0266697%2Ctt2119532%2Ctt0892769%2Ctt2267998%2Ctt0758758%2Ctt0405508%2Ctt1856101%2Ctt1979320%2Ctt2024544%2Ctt0405159%2Ctt1392190%2Ctt3315342%2Ctt1895587%2Ctt1392214%2Ctt0264464%2Ctt1201607%2Ctt1280558%2Ctt0088247%2Ctt0440963%2Ctt0325980%2Ctt5463162%2Ctt3606756%2Ctt5220122%2Ctt3766354%2Ctt4912910%2Ctt4123430%2Ctt1024648%2Ctt1504320%2Ctt1010048%2Ctt0120338%2Ctt0066763%2Ctt0079221%2Ctt2283748%2Ctt1954470%2Ctt0242519%2Ctt0374887%2Ctt0292490%2Ctt0200087%2Ctt4849438%2Ctt2377938%2Ctt0220832%2Ctt2631186%2Ctt0499375%2Ctt2883512%2Ctt0425637%2Ctt2911666%2Ctt0884732%2Ctt0455944%2Ctt0373074%2Ctt1600196%2Ctt1631867%2Ctt2397535%2Ctt2872732%2Ctt1170358%2Ctt2310332%2Ctt1809398%2Ctt1877832%2Ctt2338151%2Ctt1409024%2Ctt2245084%2Ctt3659388%2Ctt3205376%2Ctt2652092%2Ctt1798709%2Ctt1872194%2Ctt1790864%2Ctt0790724%2Ctt2562232%2Ctt1646971%2Ctt2866360%2Ctt2713180%2Ctt0829150%2Ctt2582846%2Ctt1091191%2Ctt0296572%2Ctt2084970%2Ctt2170593%2Ctt0118971%2Ctt2692250%2Ctt1392170%2Ctt2446042%2Ctt0369610%2Ctt2103281%2Ctt0808506%2Ctt4651520%2Ctt1323594%2Ctt0454921%2Ctt0787474%2Ctt0238380%2Ctt1667889%2Ctt1921149%2Ctt1865505%2Ctt3682448%2Ctt4046784%2Ctt3385516%2Ctt1440129%2Ctt0808151%2Ctt0486655%2Ctt3397884%2Ctt1772264%2Ctt0840361%2Ctt1951266%2Ctt1800241%2Ctt1355631%2Ctt0465602%2Ctt0903624%2Ctt0186151%2Ctt4178092%2Ctt2820852%2Ctt0478970%2Ctt0406375%2Ctt2199571%2Ctt3848892%2Ctt1037705%2Ctt2937898%2Ctt0787524%2Ctt1951265%2Ctt3079380%2Ctt1490017%2Ctt1253863%2Ctt0804497%2Ctt0837562%2Ctt1690953%2Ctt1731141%2Ctt0397892%2Ctt2726560%2Ctt3501416%2Ctt0206634%2Ctt0796366%2Ctt0453467%2Ctt0473075%2Ctt2305051%2Ctt1216491%2Ctt1981115%2Ctt2872750%2Ctt1454468%2Ctt0831387%2Ctt2379713%2Ctt1032751%2Ctt1823672%2Ctt0443706%2Ctt1583421%2Ctt0245844%2Ctt2224026%2Ctt2494280%2Ctt1386703%2Ctt1340138%2Ctt1663202%2Ctt1060277%2Ctt2024469%2Ctt2802144%2Ctt0109686%2Ctt2039393%2Ctt3460252%2Ctt2179136%2Ctt1596363%2Ctt2357291%2Ctt3799694%2Ctt1899353%2Ctt0499549%2Ctt2582496%2Ctt1343727%2Ctt0372183%2Ctt0217505%2Ctt3447364%2Ctt1220719%2Ctt0127357%2Ctt1791528%2Ctt2888046%2Ctt2381249%2Ctt1645170%2Ctt0241527%2Ctt0304141%2Ctt0295297%2Ctt0373889%2Ctt0330373%2Ctt2709768%2Ctt0417741%2Ctt0242653%2Ctt0234215%2Ctt1386932%2Ctt1126590%2Ctt2059255%2Ctt1457767%2Ctt2488496%2Ctt2848292%2Ctt3065204%2Ctt2452042%2Ctt3235888%2Ctt2908446%2Ctt3040964%2Ctt3488710%2Ctt2948356%2Ctt0420293%2Ctt4263482%2Ctt1124037%2Ctt4319698%2Ctt3531824%2Ctt4136084%2Ctt3521164%2Ctt4550098%2Ctt2140479%2Ctt3076658%2Ctt2140465%2Ctt4172430%2Ctt2005151%2Ctt3631112%2Ctt4034354%2Ctt2404435%2Ctt2277860%2Ctt1355644%2Ctt4624424%2Ctt4786282%2Ctt4196776%2Ctt1216475%2Ctt0317219%2Ctt2543164%2Ctt2267968%2Ctt2510894%2Ctt2788732%2Ctt4160708%2Ctt0382932%2Ctt0371746%2Ctt2975590%2Ctt3393786%2Ctt1300854%2Ctt1228705%2Ctt2660888%2Ctt0800369%2Ctt0770828%2Ctt1408101%2Ctt0317705%2Ctt1270798%2Ctt0988045%2Ctt0120915%2Ctt1130884%2Ctt0926084%2Ctt0258463%2Ctt0416449%2Ctt0126029%2Ctt1217209%2Ctt0121765%2Ctt0096874%2Ctt1250777%2Ctt1935859%2Ctt0120363%2Ctt0120903%2Ctt1430132%2Ctt0240772%2Ctt0383574%2Ctt0381061%2Ctt0936501%2Ctt0097576%2Ctt0119654%2Ctt1179933%2Ctt1843866%2Ctt4572514%2Ctt1860357%2Ctt4302938%2Ctt3748528%2Ctt0121766%2Ctt1979388%2Ctt3183660%2Ctt0349903%2Ctt0441773%2Ctt1981677%2Ctt0079945%2Ctt0084726%2Ctt0088170%2Ctt0102975%2Ctt0098382%2Ctt0092007%2Ctt0117731%2Ctt0253754%2Ctt0120844%2Ctt0111280%2Ctt3416828%2Ctt0268380%2Ctt0486822%2Ctt1454029%2Ctt1679335%2Ctt1473832%2Ctt5613056%2Ctt3691740%2Ctt3890160%2Ctt0891527%2Ctt4361050%2Ctt4430212%2Ctt4190530%2Ctt3721936%2Ctt4698684%2Ctt0248667%2Ctt2582782%2Ctt4169250%2Ctt3263904%2Ctt1951264%2Ctt2294629%2Ctt2381111%2Ctt2513074%2Ctt1638355%2Ctt3464902%2Ctt3011894%2Ctt3640424%2Ctt1619029%2Ctt0115759%2Ctt4425200%2Ctt5362988%2Ctt4276820%2Ctt0790636%2Ctt0437800%2Ctt0451279%2Ctt1670345%2Ctt1961175%2Ctt0378194%2Ctt3110958%2Ctt1959563%2Ctt1615160%2Ctt0189584%2Ctt0120586%2Ctt0112471%2Ctt3203606%2Ctt0190332%2Ctt4649466%2Ctt1065073%2Ctt1259528%2Ctt0112573%2Ctt5397194%2Ctt2980516%2Ctt3411444%2Ctt1298650%2Ctt1590193%2Ctt0112453%2Ctt2547584%2Ctt4555426%2Ctt4062536%2Ctt3416532%2Ctt4846340%2Ctt3606752%2Ctt4385888%2Ctt3387266%2Ctt4034228%2Ctt4540710%2Ctt3470600%2Ctt5052448%2Ctt2671706%2Ctt3741834%2Ctt3666024%2Ctt0449088%2Ctt2167266%2Ctt2852458%2Ctt2350496%2Ctt2203308%2Ctt3169706%2Ctt1355683%2Ctt3322364%2Ctt3792960%2Ctt1620935%2Ctt1878870%2Ctt4209788%2Ctt0244316%2Ctt0485947%2Ctt0480669%2Ctt1258197%2Ctt1148204%2Ctt1276104%2Ctt0421715%2Ctt0464141%2Ctt1139797%2Ctt1182345%2Ctt1588173%2Ctt0448134%2Ctt0050083%2Ctt1255953%2Ctt1305806%2Ctt1074638%2Ctt0081505%2Ctt0443543%2Ctt1229238%2Ctt0364569%2Ctt4972582%2Ctt0338013%2Ctt0366548%2Ctt0983193%2Ctt1515091%2Ctt0298148%2Ctt0472043%2Ctt1365519%2Ctt1477834%2Ctt5095030%2Ctt4881806%2Ctt0299658%2Ctt0138097%2Ctt0105695%2Ctt0099348%2Ctt0097239%2Ctt0095953%2Ctt0091763%2Ctt0093389%2Ctt6053438%2Ctt6499752%2Ctt7784604%2Ctt7668870%2Ctt7040874%2Ctt4575576%2Ctt2119543%2Ctt6257174%2Ctt3750872%2Ctt3892172%2Ctt7336572%2Ctt6182908%2Ctt3104988%2Ctt7349662%2Ctt1517451%2Ctt5052474%2Ctt1691917%2Ctt2980706%2Ctt6566624%2Ctt7014006%2Ctt7890060%2Ctt5688932%2Ctt6679794%2Ctt4244998%2Ctt1213641%2Ctt1270797%2Ctt6850820%2Ctt1502407%2Ctt2388771%2Ctt2737304%2Ctt3759298%2Ctt1846589%2Ctt4701182%2Ctt1727824%2Ctt4218572%2Ctt0245429%2Ctt0120382%2Ctt0457430%2Ctt0414993%2Ctt0139809%2Ctt0390384%2Ctt0092563%2Ctt0119174%2Ctt1191111%2Ctt0166924%2Ctt4530422%2Ctt5848272%2Ctt5083738%2Ctt6343314%2Ctt1289217%2Ctt4633694%2Ctt7401588%2Ctt6966692%2Ctt4154664%2Ctt4154796%2Ctt0437086%2Ctt2386490%2Ctt2325846%2Ctt6513120%2Ctt7775622%2Ctt6155172%2Ctt8130968%2Ctt6491178%2Ctt7959026%2Ctt1860242%2Ctt0448115%2Ctt6823368%2Ctt6857112%2Ctt5028340%2Ctt6472976%2Ctt5461944%2Ctt4669788%2Ctt5886046%2Ctt5932728%2Ctt2481498%2Ctt6146586%2Ctt4943998%2Ctt1489887%2Ctt5968394%2Ctt6292852%2Ctt1979376%2Ctt5884052%2Ctt5027774%2Ctt1080016%2Ctt0438097%2Ctt0351283%2Ctt0479952%2Ctt1277953%2Ctt0087469%2Ctt0367882%2Ctt0082971%2Ctt6320628%2Ctt8772262%2Ctt6139732%2Ctt6565702%2Ctt7458762%2Ctt0073486%2Ctt0317248%2Ctt0038650%2Ctt0118799%2Ctt0120689%2Ctt0054215%2Ctt0110413%2Ctt0253474%2Ctt0110357%2Ctt0078788%2Ctt0887912%2Ctt0375679%2Ctt0119488%2Ctt6921996%2Ctt0274166%2Ctt1634122%2Ctt0301196%2Ctt0337978%2Ctt0112864%2Ctt0376994%2Ctt0458525%2Ctt0290334%2Ctt0075314%2Ctt5164214%2Ctt0317919%2Ctt0117060%2Ctt0120755%2Ctt0181689%2Ctt6105098%2Ctt1453405%2Ctt0120131%2Ctt2139881%2Ctt8752440%2Ctt8022928%2Ctt3361792%2Ctt6348138%2Ctt2066051%2Ctt3861390%2Ctt7349950%2Ctt6806448%2Ctt7286456%2Ctt2935510%2Ctt6324278%2Ctt9243946%2Ctt7131622%2Ctt6398184%2Ctt8079248%2Ctt6189022%2Ctt0477347%2Ctt1078912%2Ctt1436562%2Ctt3469046%2Ctt0099088%2Ctt0456144%2Ctt0113497%2Ctt6645614%2Ctt0181852%2Ctt0438488%2Ctt0381681%2Ctt2209418%2Ctt0099423%2Ctt1606378%2Ctt0086154%2Ctt0091799%2Ctt0103644%2Ctt0118583%2Ctt1194173%2Ctt0120912%2Ctt0099674%2Ctt1790809%2Ctt1402488%2Ctt1318514%2Ctt0413267%2Ctt0892791%2Ctt1772341%2Ctt1302011%2Ctt1046173%2Ctt1397280%2Ctt3410834%2Ctt1840309%2Ctt0322259%2Ctt0463985%2Ctt0232500%2Ctt1596343%2Ctt1905041%2Ctt1013752%2Ctt4630562%2Ctt0134847%2Ctt1411250%2Ctt6359956%2Ctt0107290%2Ctt0119567%2Ctt0163025%2Ctt4765284%2Ctt1694542%2Ctt0070328%2Ctt0055928%2Ctt0057076%2Ctt0058150%2Ctt0059800%2Ctt0062512%2Ctt0064757%2Ctt0066995%2Ctt0071807%2Ctt0076752%2Ctt0079574%2Ctt0082398%2Ctt0086034%2Ctt0090264%2Ctt0097742%2Ctt0113189%2Ctt0120347%2Ctt0830515%2Ctt0143145%2Ctt0246460%2Ctt0093428%2Ctt0086006%2Ctt0169102%2Ctt0367110%2Ctt2356180%2Ctt4934950%2Ctt5982852%2Ctt2621000%2Ctt4068576%2Ctt8526872%2Ctt7984766%2Ctt4364194%2Ctt7343762%2Ctt4729430%2Ctt2279373%2Ctt7798634%2Ctt8096832%2Ctt1302006%2Ctt5606664%2Ctt6259380
2019-12-18 11:39 WARNING  task          watchlist_local Aborting task (plugin: imdb_watchlist)
```
</details>

### Additional information:

- FlexGet version: 3.0.16
- Python version: Python 3.7.5
- Installation method: docker (wiserain/flexget)

**Comment by kxzaon** [comment]
Same issue with Flexget 3.1.51. Anyone for a solution ? Even for a dirty one ? Because at this time, even the imdb_list plugin doesn't work...
FYI, I've got 800 titles in my imdb watchlist.
Thanks a lot for your help.

**Comment by KronK0321** [comment]
I ran into this same issue recently. Likewise, my imdb watchlist had 800+ entries.

My workaround was to create a new list and manually move about 200 movies from my watchlist to the 2nd list. Once I got down to about 615 entries, the task that populated the local movie list from the imdb list started working again.

Then I split my single task into two separate tasks:
```
  populate_movie_list_jordan:
    imdb_watchlist:
      user_id: '{? imdb.user_id ?}'
      list: watchlist
    accept_all: yes
    list_add:
      - movie_list: imdb_movies

  populate_movie_list2_jordan:
    imdb_watchlist:
      user_id: '{? imdb.user_id ?}'
      list: ls086454039    #not my actual list
    accept_all: yes
    list_add:
      - movie_list: imdb_movies
```

This adequately works around the issue for me. Obviously it's a bit hack-ish and will begin to fail if my 2nd imdb list approaches 615+ entries. But it's the only solution I've found without modifying flexget source code.
